frontend/src/components/AppPercentage.vue

```diff
@@ -37,6 +55,8 @@ const d = computed(() => {
   const y = -cos(angle) * 50;
   return `M 0 -50 A 50 50 0 ${angle >= 180 ? 1 : 0} 1 ${x} ${y}`;
 });
+
```

Comment:
I don't think this needs to be clamped at all. Also, nitpicking, but when the computation is this simple, and not repeated anywhere, I feel you might as well just put it directly in the `<template>`, no need for `computed` property. Vue style guides do recommend that you use computed properties as much as possible to keep the template simple, but I feel this can really be taken to an extreme.